rules/sources/contact_data.yaml

```diff
@@ -25,7 +25,7 @@ sources:
     isSensitive: False
     sensitivity: medium
     patterns:
-      - "(?i)((phone|mobile|cellphone|contact)|(.*((phone|mobile|cellphone|contact)[^\\s/(;)#|,=!>]{0,2}(?:detail|details)|(?:mobile|phone|cellphone|contact)[^\\s/(;)#|,=!>]{0,10}(?:number|nbr|num|no)|(?:primary|secondary)[^\\s/(;)#|,=!>]{0,10}(phone|landline|telephone))))"
+      - "(?i)((phone|mobile|cellphone|contact|work)|(.*((phone|mobile|cellphone|contact)[^\\s/(;)#|,=!>]{0,2}(?:detail|details)|(?:mobile|phone|cellphone|contact)[^\\s/(;)#|,=!>]{0,10}(?:number|nbr|num|no)|(?:primary|secondary)[^\\s/(;)#|,=!>]{0,10}(phone|landline|telephone))))"
```

Comment:
Do we intend to match `work` in  `ContactData.Phonenumber`?

Reply:
Only 'workPhone' , this wasn't getting matched by previous regex.

Reply:
@khemrajrathore should I change this regex to be something else ?

Reply:
If we intent to match `workPhone` update the regex from `work` to `workPhone` as with current regex `workphone` won't get matched unless we explicitly add a `.*`

Reply:
With current regex, `workPhone` does match, can you take a look again ?